classes/submission/maps/Schema.php

```diff
@@ -144,21 +195,50 @@ public function summarizeMany(Enumerable $collection, LazyCollection $userGroups
         $this->collection = $collection;
         $this->userGroups = $userGroups;
         $this->genres = $genres;
-        return $collection->map(function ($item) {
-            return $this->summarize($item, $this->userGroups, $this->genres);
-        });
+        $this->reviewAssignments = Repo::reviewAssignment()->getCollector()->filterBySubmissionIds($collection->keys()->toArray())->getMany()->remember();
```

Comment:
Here you are using the `remember()` function while in other places in the same file you are not (like `Repo::reviewAssignment()->getCollector()->filterBySubmissionIds([$item->getId()])->getMany()`). Is this indented?

Reply:
Thanks, missed that!